pramen-py/src/pramen_py/test_utils/fixtures.py

```diff
@@ -39,7 +39,7 @@
 
 env = Env(expand_vars=True)
 env.read_env(
-    path=(pathlib.Path(".") / ".env").as_posix(),
+    path=(pathlib.Path(".") / ".env.test").as_posix(),
```

Comment:
I think it is better to keep the tests and the app referring the .env file. And Makefile is responsible to symlink the correct .env.${ENV} to it. See https://github.com/AbsaOSS/pramen/blob/main/pramen-py/Makefile#L15. 

But I think you're right that having the .env.local in the repo without encrypting it is a good idea. So please, just rename .env.test -> .env.local and run once make test. It will symlink it to .env and your local environment will be ok

Reply:
Understand. But .env.local exists only for the purpose of the local development environment. So let's then either update the Makefile to default to .env.test, or rename .env.test -> .env.local in your MR and call it explicitly in the tests. What do you think?

Reply:
We can distinguish between the local dev environment and the unit test environment. For instance,
- `.env.local` can point to a local Hadoop installation and a Spark cluster installed in standalone mode
- `.env.test` would still point to local filesystem and Spark in local mode.